components/nodes/embeds/Airtable.js

```diff
@@ -0,0 +1,21 @@
+export default function Airtable({ node, amp }) {
+  //this is to add the 'embed' before the id since that's how airtable set it up
+  const partial_src = node.slice(0, 21);
+  const id = node.slice(21);
```

Comment:
I think this will work, but I don't want to rely on the length of the string. Instead, let's use [split](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split) and use `/` as the character to split on. You should be able to isolate the ID this way.